source/materials/OpticalMaterialProperties.cc

```diff
@@ -18,7 +18,7 @@ using namespace nexus;
 using namespace CLHEP;
```

Comment:
It doesn't let me add a comment above, but it looks like we don't need `assert.h` on line 15. I tried to comment it out and it compiles and runs fine, can you double check?

Reply:
Correct! I eliminated it.